graph/articles/collections.md

```diff
@@ -316,3 +316,148 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
+
+Collections of entity types are generally preferable to collections of structual types because collections of structural types must be updated as a single unit, meaning that they are overwritten entirely by new contents, rather than be updated relative to the existing contents. 
+Sometimes, structural collection properties are added to a type and then scenarios are discovered later that require a collection of entity types.
+Take the following model with an entity type `foo` that has a collection of `bar`s:
+
+```xml
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
+</EntityType>
+
+<ComplexType Name="bar">
+  <Property Name="prop1" Type="Edm.String" />
+  <Property Name="prop2" Type="Edm.String" />
+</ComplexType>
+```
+and a scenario arises that requires, for example, to remove individual `bar`s from the collection. 
+There are two options forward: //// TODO do we want to offer both options, or just one?
+
+### 11.1 Side-by-side collection properties
+
+The model can be updated to have two collections side-by-side:
+```diff
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
++ <NavigationProperty Name="barsAsEntities" Type="Collection(self.barAsEntity)" ContainsTarget="true" />
+</EntityType>
+
+<ComplexType Name="bar">
+  <Property Name="prop1" Type="Edm.String" />
+  <Property Name="prop2" Type="Edm.String" />
+</ComplexType>
+
++<EntityType Name="barAsEntity">
++ <Key>
++   <PropertyRef Name="prop1" />
++ </Key>
++ <Property Name="prop1" Type="Edm.String" />
++ <Property Name="prop2" Type="Edm.String" />
++</EntityType>
+```
+Clients will now be able to refer to individual `bar`s using `prop1` as a key, and they can now remove those `bar`s using `DELETE` requests:
+```http
+DELETE /foos/{fooId}/bars/{some_prop1}
+```
+```http
+HTTP/1.1 204 No Content
+```
+The expectation is that `bars` and `barsAsEntities` are treated as two "views" into the same data.
+To meet this expectation, workloads must:
```

Comment:
I didn't understand why we would enforce ambiguity of updating two places indefinitely?  I think that we need to deprecate wrong modeling choice and have a single option to treat so called 'bars'.

Reply:
I'm happy to add a sentence that the old property should be marked deprecated. I do think these expectations need to be honored at least while both properties exist side-by-side. I personally don't think we need to drive workloads to actually remove the property, their own supportability overhead should handle that if necessary.

Do you agree that these expectations need to be met while we have both properties? If so, does it work for you if I just add a sentence to mark the old property as deprecated?